Sources/Widgets/Widgets3D/SplineWidget/index.js

```diff
@@ -89,4 +89,4 @@ export const newInstance = macro.newInstance(extend, 'vtkSplineWidget');
 
 // ----------------------------------------------------------------------------
 
-export default { newInstance, extend };
+export default { newInstance, extend, widgetBehavior };
```

Comment:
you do not need to export this one... if you want it, you simply import the behavior.js file